.github/chainguard/lofo.sts.yaml

```diff
@@ -2,7 +2,11 @@
 
 issuer: https://accounts.google.com
 
-subject: "107094302893801739313"
+
+# 107094302893801739313 -
```

Comment:
whats this one for?

Reply:
Not sure, it existed previously. The only other person using this service is @joshrwolf (that I know of) but I looked in his project and didn't see it. I'm waiting until he's back online to address it